docs/administration/segregation.md

```diff
@@ -67,7 +67,26 @@ To apply a default marking when creating a new entity or relationship, you can c
     Simply adding markings as default markings is insufficient to display the markings when creating an entity or relationship. You also need to enable default markings in the customization settings of an entity or relationship. For example, to enable default markings for a new report, navigate to "Settings > Customization > Report > Markings" and toggle the option to `Activate/Desactivate default values`.
 
 
-## Management of multiple markings
+### Maximum shareable marking definitions
+
+This configuration allows to define, for each type of marking definitions, until which level
+we allow to share data externally (via Public dashboard or file export).
+
+The marking definitions that can be shared by a group are the ones
```

Comment:
Need to leave a blank line before the first point of the list otherwise the list won't be properly formatted

Reply:
fixed